cmd/hw001/_experimental/exp-hw-001.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"github.com/kyokomi/emoji"
+	helloWorldPrinter "github.com/rodkevich/go-course/homework/hw001/hwp"
+)
+
+func main() {
+	// using bytes.Buffer
+	var w = fmt.Println
+	helloWorldPrinter.AddEmoji(`:relaxed:`, w)
```

Comment:
Great experiment!!
some remarks
it is better to implement io.Writer or io.StringWriter in you package
then it will looks like this
```suggestion
	var w io.StringWriter = hwp.New(hwp.ModeBuffer) 
	_, err := w. WriteString(`:relaxed:`)
	...
```